spinoso-time/Cargo.toml

```
@@ -13,9 +13,17 @@ license = "MIT"
 keywords = ["artichoke", "date", "datetime", "spinoso", "time"]
 categories = ["date-and-time"]
 
+[features]
```

Can we enable both backends by default like the rest of the spinoso crates do?

```suggestion
[features]
default-features = ["chrono", "tzrs"]
```

e.g. `spinoso-array`:

https://github.com/artichoke/artichoke/blob/06e9e0d88dc7d7451dc5b2269e9d05c69bef5da6/spinoso-array/Cargo.toml#L26

typically, spinoso crates have a "default" backend that is always available even with `default-features = false`, but that seems fine to not follow here since this crate is still in flux.

---

Done! Assuming you did mean `default-features` and not just `default` right?